docs/user/filesystem/commandline.rst

```diff
@@ -41,6 +41,39 @@ Argument                 Description
 
 The ``--temp-path`` argument may be useful when working with ``s3`` data, as ``s3`` is slow for incremental writes.
 
+``generate-partition-filters``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Calculate filters that exactly match partitions. This can be used to facilitate exports from another system
+directly into the appropriate partition directory.
+
+======================== =========================================================
+Argument                 Description
+======================== =========================================================
+``-p, --path *``         The filesystem root path used to store data
+``-f, --feature-name *`` The name of the schema
+``-q, --cql``            CQL predicate to determine the partitions to operate on
+``--partitions``         Partitions to operate on, subject to the CQL filter
+``--no-header``          Suppress the column headers in the output
+``--config``             Hadoop configuration properties, in the form k=v
```

Comment:
Some note; I wonder if it'd be better to say "key=value" instead.

Reply:
updated